server/form-pages/assess/assessApplication/suitablityAssessment/suitabilityAssessment.ts

```diff
@@ -0,0 +1,104 @@
+import type { TaskListErrors, YesOrNo } from '@approved-premises/ui'
+
+import { Page } from '../../../utils/decorators'
+import { sentenceCase } from '../../../../utils/utils'
+
+import TasklistPage from '../../../tasklistPage'
+
+export type SuitabilityAssessmentSection = {
+  id: 'riskFactors' | 'riskManagement' | 'locationOfPlacement' | 'moveOnPlan'
+  question: string
+}
+
+@Page({
+  name: 'suitability-assessment',
+  bodyProperties: [
+    'riskFactors',
+    'riskFactorsComments',
+    'riskManagement',
+    'riskManagementComments',
+    'locationOfPlacement',
+    'locationOfPlacementComments',
+    'moveOnPlan',
+    'moveOnPlanComments',
+  ],
+})
+export default class SuitabilityAssessment implements TasklistPage {
+  name = 'suitability-assessment'
+
+  title = 'Suitability assessment'
+
+  sections: Array<SuitabilityAssessmentSection> = [
```

Comment:
One more improvement (sorry!). Could sections just be a key/value record now? So:

```
sections = {
  riskFactors: 'Does the application identify the risk factors that an Approved Premises (AP) placement can support?',
...
}
```